src/main/java/com/epam/lowcost/service/UserServiceImpl.java

```diff
@@ -37,4 +37,11 @@ public User updateUser(User user) {
     public String deleteUser(long userId) {
         return userDAO.deleteUser(userId);
     }
+
+    @Override
+    public User verifyUser(String log, String pass) {
```

Comment:
Has user field log, pass ? Give appropriate names for variables

Reply:
Done